src/utils/discordUtil.ts

```diff
@@ -73,11 +76,11 @@ export const fetchPrefix = async (message: Message) => {
 export const purge_messages = async (message: Message, number_of_msgs: number) => {
   if (message.channel.type == 'GUILD_TEXT') {
     const fetchedMessages = await message.channel.messages.fetch({limit: number_of_msgs, before: message.id});
-    const messagesToDelete = await fetchedMessages.filter((m: Message) => !m.pinned);
+    const messagesToDelete = await fetchedMessages.filter((m: Message) => !m.pinned || m.createdTimestamp < (message.createdTimestamp - fourteenDaysInMs));
     await message.channel.bulkDelete(messagesToDelete);
-    return true;
+    return messagesToDelete.size;
   }
-  return false;
```

Comment:
I think this should still return false. In the main command, if it returns false, say a fatal error occurred. If the value is 0, you can say that no messages were found

Reply:
You can check if it is literally false rather than coercing it into a boolean to make this differentiation